### PR TITLE
Update help command (affects UI)

### LINKED ISSUE #331: Bug - Help description incomplete (impacts UI)

### PR DESCRIPTION
**Pull request by jianyangg**
Also updated UG change of name for command
summary

Close #331

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/HelpCommand.java [hunk 1]
```diff
@@ -10,30 +10,49 @@ public class HelpCommand extends Command {
...
-        + "More commands will be added in the future.";
+            + " in before using the following basic commands, except for the ones indicated with a `.\n\n"
+            + "USER\n"
+            + "These are commands for managing your user account in HomeBoss.\n"
+            + "* `register - Registers a new user account to use HomeBoss.\n"
+            + "* `login - Logs in to your user account.\n"
+            + "* update - Updates your user account details.\n"
+            + "* `recover account - Recovers your user account.\n"
+            + "* `logout - Logs out of your user account.\n"
+            + "* `delete account - Deletes your user account.\n\n"
```
Comment (juliusgambe) [inline src/main/java/seedu/address/logic/commands/HelpCommand.java:26]: Why are there some that starts with ` and doesn't end with ` ? Copy paste error?

**Review by juliusgambe — COMMENTED** [review]
I think there was a transfer error from User Guide to help

File: src/main/java/seedu/address/logic/commands/HelpCommand.java (hunk 1 above)
Comment (jianyangg) [reply src/main/java/seedu/address/logic/commands/HelpCommand.java:26]: The comment at the top of the help message explains it; These were meant to differentiate between commands that can be used before logged in / account is registered and the rest

**Review by juliusgambe — APPROVED** [review]
LGTM

**Pull request merged**